Mod1/ProfessionalDevelopment/PeerFeedback.md

```diff
@@ -0,0 +1,46 @@
+# Peer Code Feedback
+
+<!-- Instructor note, try to pair up students of roughly similar skill level for giving feedback. -->
+
+As with writing an essay, having an outside set of eyes give feedback can be very helpful for improving the quality of the end product. 
+Additionally, reading other people's code is one of the best ways to level up as a developer!
+
+For these reasons, code review is a key part of working on most development teams and something we will practice starting today.
+
+You will write your notes using [this google doc template](https://docs.google.com/document/d/1nJMCJT5yQcu-mLV_ONA906QOCJxIUrsIgUobeKFTyF4/copy) and then share the doc with your partner.
+
+For part 1, which takes place this afternoon, you and your partner will exchange Github links for your project repos and then independently walk through the Setup section and write usability feedback. We recommend spending 15 to 30 minutes on this portion.
+
+For part 2, which will take place tomorrow morning, you will be in a breakout room with your partner for 50 minutes. Each partner will have 20 minutes to share their screen and talk through the sections of code they would most like feedback on. Make sure to show your tests. The partner giving feedback will ask questions and take notes for the Code Structure, Testing, and Additional Feedback and Notes sections of the google doc. Next, take 5 minutes to share your feedback. Then switch places and repeat the process where the other partner is sharing their screen.
+
+*Remember that feedback should be actionable and kind.*
+
+## Part 1: Complete this portion separately before you meet with your partner tomorrow.
+
+### Setup
+First,
+* Clone down your partner’s project
```

Comment:
Should we call out that they do not need to fork before cloning?

Reply:
Ooo very good clarification! Will edit.